connector_algolia/models/se_backend_algolia.py

```diff
@@ -13,19 +12,14 @@ class SeBackendAlgolia(models.Model):
     _inherit = 'se.backend.spec.abstract'
     _description = 'Algolia Backend'
 
-    algolia_app_id = fields.Char(sparse="data", string="APP ID")
-    algolia_api_key = fields.Char(related='password', string="API KEY")
+    # TODO: load values from server env
+    algolia_app_id = fields.Char(string="APP ID")
+    # v12: we removed keychain inheritance
```

Comment:
Note sure of the result of oldname as the field password was a computed field, so nothing in the database.
What do you think?

Reply:
hmm being a stored field it should work.
I have no proof tho, so what we can do is: test it and add a migration step if needed.
I'd say: let's add a TODO and maybe mention it in the ROADMAP.
Then you can do the test w/ your old instances and see what happens.
We might have to add other migration code anyway... who knows? :wink: 
Fine?

Reply:
Adding an explication in the README will be enough.
If people remove keychain, (because they use server_env or something else) they will store the password elsewhere so they will to handle this (it's new configuration). If you still use keychain (with some extra glue module) it will work as before